mithril-signer/src/database/query/protocol_initializer/get_protocol_initializer.rs

```diff
@@ -0,0 +1,53 @@
+use sqlite::Value;
+
+use mithril_common::entities::Epoch;
+use mithril_persistence::sqlite::{Query, SourceAlias, SqLiteEntity, WhereCondition};
+
+use crate::database::record::ProtocolInitializerRecord;
+
+/// Simple queries to retrieve [ProtocolInitializer] from the sqlite database.
+pub struct GetProtocolInitializerQuery {
+    condition: WhereCondition,
+    limit: Option<usize>,
+}
+
+impl GetProtocolInitializerQuery {
+    /// Get all signed beacons that match the given signed entity types.
```

Comment:
This comment seems to be outdated

Reply:
Replace by : /// Get protocol initializer that match the epoch.